packages/tabs/src/tabs-overflow.css

```diff
@@ -10,6 +10,8 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+@import './spectrum-tabs-overflow.css';
```

Comment:
If you end up sharing, you'll want to import this in the JS instead of the CSS so that it doesn't get compiled into two different files at delivery time.